Requirements/base.txt

```diff
@@ -0,0 +1,17 @@
+aspy.yaml==1.0.0
```

Comment:
Most of the the packages here are indirect, recall ones you installed explicitly and remove the rest of them.